Northstar.CustomServers/mod/scripts/vscripts/earn_meter/sv_earn_meter_mp.gnut

```diff
@@ -91,6 +91,8 @@ void function EarnMeterMP_PlayerLifeThink( entity player )
 	EarnObject pilotReward = PlayerEarnMeter_GetReward( player ) 
 	float pilotRewardFrac = PlayerEarnMeter_GetRewardFrac( player )
 	int lastEarnMeterMode = PlayerEarnMeter_GetMode( player )
+	bool saidTitanSoon = false
+	bool titanReady = false
```

Comment:
Should this not technically be saidTitanReady,  Hardly matters though

Reply:
No because `titanReady` is a variable that displays the Titan message, not to make Faction Leader speak about it.

Reply:
Should still be something like `messagedTitanReady` or something as the variable doesn't per se track if the Titan is ready but whether the message saying "Titan ready" was sent.

Reply:
Done